record/record.app.js

```diff
@@ -46,7 +48,12 @@
         $rootScope.showDeleteButton = chaiseConfig.deleteRecord === true ? true : false;
 
         try {
-            var ermrestUri = UriUtils.chaiseURItoErmrestURI($window.location);
+            // TODO: Redirecting to home page is appropriate for an error in chaiseURItoErmrestURI
+            // try {
+                var ermrestUri = UriUtils.chaiseURItoErmrestURI($window.location);
+            // } catch (e) {
+            //     ErrorService.errorPopup(exception.message, exception.code, "home page");
```

Comment:
os should this throw a malformed URI exception that is then caught by the outer try/catch?